equinoxfitness/datacoco-email_tools
# Feature/update tests

## Description

Updated tests to increase coverage


## Changes

- add more tests
- update pre-commit-config
- add pypi badge

### How Has This Been Tested?

1. Checkout this branch

1. Switch to virtual environment

1. Run `python -m unittest tests.test_email_tools`

- [ ] Verify that all 6 tests will be successful
